src/nodejs/neo-rpc/v2.9.0/client/module/getNodesBy.js

```diff
@@ -83,17 +83,17 @@ exports.getRpcNodes = (options) => {
 
 exports.ping = (options) => {
   let rankedList = [], i = 0
-  let opts = options ? options : opts = {}
-
-  opts.net ? opts.net : opts.net = 'TestNet'
-  opts.order ? opts.order : opts.order = 'asc'
-
-  let net    = netUtil.resolveNetworkId(opts.net)
-  let cfgNodes  = cfg.getNodes(net)
-
-  opts.nodes ? opts.nodes : opts.nodes = cfgNodes
-
-  let nodes = opts.nodes
+  // let opts = options ? options : opts = {}
```

Comment:
I'm leaving these here for now as I still haven't decided if I want to do this here or in the calling module. Thoughts?